lib/bloc/podcast_history/sqflite/podcast_history_database.dart

```diff
@@ -0,0 +1,70 @@
+import 'dart:developer';
+
+import 'package:breez/bloc/podcast_history/sqflite/podcast_history_model.dart';
+import 'package:sqflite/sqflite.dart';
+import '../../../logger.dart';
+import '../../pos_catalog/sqlite/db.dart';
+
+class PodcastHistoryDatabase {
+  static final PodcastHistoryDatabase instance = PodcastHistoryDatabase._init();
+
+  static Database _database;
+  PodcastHistoryDatabase._init();
+
+  Future<Database> get database async {
+    if (_database != null) {
+      return _database;
+    }
+    _database = await _initDB("podcast_history.db");
+    return _database;
+  }
+
+  Future<Database> _initDB(String filePath) async {
+    final dbPath = await getDatabasePath();
+    final path = dbPath + filePath;
+    return await openDatabase(path, version: 1, onCreate: _createDB);
+  }
+
+  Future _createDB(Database db, int version) async {
+    final fieldIdType = 'INTEGER PRIMARY KEY AUTOINCREMENT';
```

Comment:
Not sure these constants help make the code more readable. It seems easier to follow a complete sql statement string.

Reply:
@roeierez Something like this?

```
 Future _createDB(Database db, int version) async {
    await db.execute('''
CREATE TABLE $podcastHistoryTable(
${PodcastHistoryFields.fieldId} INTEGER PRIMARY KEY AUTOINCREMENT,
${PodcastHistoryFields.podcastId} TEXT NOT NULL,
${PodcastHistoryFields.timeStamp} TEXT NOT NULL,
${PodcastHistoryFields.satsSpent} INTEGER NOT NULL,
${PodcastHistoryFields.boostagramsSent} INTEGER NOT NULL,
${PodcastHistoryFields.podcastName} TEXT NOT NULL,
${PodcastHistoryFields.podcastImageUrl} TEXT NOT NULL,
${PodcastHistoryFields.durationInMins} DECIMAL NOT NULL
 )
 ''');
  }
```